PSKoans/Init/RegisterArgumentCompleters.ps1

```diff
@@ -4,6 +4,8 @@ $CommandName = @(
     'Reset-PSKoan'
     'Show-Karma'
     'Update-PSKoan'
+    'Show-Advice'
+    'Get-Advice'
```

Comment:
We should put the `*-Advice` command names in a different variable or redefine `$CommandName` immediately before registering the argumentcompleter below, so we don't accidentally register completers for commands they're not valid for. 🙂

Reply:
Ah, yes, that makes perfect sense. I'll get on that as soon as I can.